Makefile

```diff
@@ -29,6 +29,11 @@ tag: ## Tag the current release
 publish: ## Put the release on GitHub
 	dune-release publish distrib
 
+.PHONY: promote-ppx-output
+promote-ppx-output: ## Promotes the current output of the ppx unit tests to be the new expected output.
+	dune runtest --force; true # Without the 'true' Make will abort.
+	cp _build/default/tests/test_ppx/test_ppx.result.reformatted.ml tests/test_ppx/test_ppx.expected.ml
```

Comment:
I think it would be nicer if this weren't dependent on the CWD. I think if you could explicitly reference these paths [relative to the Makefile](https://timmurphy.org/2015/09/27/how-to-get-a-makefile-directory-path/) that would make it a bit less fragile.

Reply:
Neat trick, thanks!